src/make_plots.R

```diff
@@ -0,0 +1,66 @@
+## ---------------------------
+##
+## Script name: make_plots
+##
+## Purpose of script: selecting sites
+##
+## Author: Jack Holbrook (USGS)
+##
+## ---------------------------
+## Notes:
+##     ~
+## ---------------------------
+
+plot_tot <- function(in_monthDat, fileout)
+{
+  #convert((num(n_per_month)))
+  print(in_monthDat)
+  
+  in_monthDat <- readr::read_csv(in_monthDat)
+  #in_monthDat <- as.numeric(in_monthDat$n_per_month)
+  
+  months <- in_monthDat %>% 
+    # uses simple val 27 in absence of costly date check function
+    filter(n_per_month > 27) %>% 
+    subset(month_year < "1995-01-01") %>% 
+    group_by(seg_id_nat) %>% 
+    mutate(n_all_time= sum(n_per_month)) #%>% 
+   # mutate(months = count(month_year))
+  
+  v <- 0 
+  for (i in 1:nrow(months)) {  # nots ure why this is computing out the way it is. 
+    if (months[i, 4] > 10000)  #this is the check the amount of data avaliable. 
```

Comment:
I think you're trying to find the number of sites with >10,000 obs, but I'm seeing a few issues with this calculation.

First, you're only counting complete months (maybe that's what you want), but if you want to a site to have a lot of January observations (for example), this check won't ensure that you have enough complete Januaries to do a trend analysis. Second, above, you're grouping and mutating, which means you're maintaining all rows in your `n_all_time` calculation. This means if a site does have >10,000 observations, you'll count in `n_all_time` times, which I suspect is why it's not counting the way you expect. 

Instead, if you did a `group_by` and `summarize` to calculate `n_all_time`, then you could do `sum(months$n_all_time > 10000)` to make the same calculation.

Reply:
I will try this!

Reply:
fixed this